monai/inferers/inferer.py

```diff
@@ -13,7 +13,7 @@
 
 import warnings
 from abc import ABC, abstractmethod
-from typing import Any, Callable, Dict, Generic, Mapping, Sequence, TypeVar, Union
+from typing import Any, Callable, Mapping, Sequence
```

Comment:
Thanks for the quick fix. Is the any reason that you are not importing from `collections.abc` here?

Reply:
This was because python 3.7 didn't support subscribing those from `collections.abc` (which was done while setting the TypeVar of `Inferer` with `[Union[torch.Tensor, Sequence[torch.Tensor], Dict[Any, torch.Tensor]]]`). 
But since they are now only used in annotations, I think they can be imported from `collections.abc`.